libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -28,6 +28,7 @@ import { createComponentRenderer, createHtmlRenderer, createTemplateRenderer } f
 import { toObject } from '../utils/object/to-object';
 import { afterRenderFinished } from '../utils/render/render-delay';
 import { unreachable } from '../utils/types/unreachable';
+import { IRenderFunction } from 'office-ui-fabric-react';
```

Comment:
Please don’t import stuff from `office-ui-fabric-react` inside the `@angular-react/core` package - we want to keep it library-agnostic.

A more generic type is needed here instead therefore.